qi-lib/flow/core/compiler/deforest/cps.rkt

```diff
@@ -21,7 +21,7 @@
 (define-syntax inline-compose1
   (syntax-rules ()
     [(_ f) f]
-    [(_ [op f] rest ...) (op f (inline-compose1 rest ...))]))
+    [(_ [op (f ...)] rest ...) (op f ... (inline-compose1 rest ...))]))
```

Comment:
Ah, now this macro is changing: does this need propagated to other versions or no?

Reply:
No.